test/micro/org/openjdk/bench/java/lang/StackWalkBench.java

```diff
@@ -286,6 +340,24 @@ public void run() {
         }
     }
 
+    /**
+     * Baseline for generating a call stack of the given depth.
+     * (No StackWalk operations are executed)
+     */
+     @Benchmark
+     public void makeCallStack(Blackhole bh) {
+         final Blackhole localBH = bh;
+         final boolean[] done = {false};
+         new TestStack(depth, new Runnable() {
+             public void run() {
+                 done[0] = true;
+             }
+         }).start();
+         if (!done[0]) {
+             throw new RuntimeException();
+         }
+     }
+
```

Comment:
I'm fine having this benchmark active by default, though offline we had discussed adding it commented out.

Reply:
Having a second thought, I keep this active as that would be a good reference number for other benchmarks as they all build test stack.